Zotero/Scenes/Detail/PDF/Views/PDFReaderViewController.swift

```diff
@@ -411,9 +411,9 @@ class PDFReaderViewController: UIViewController {
             viewModel.process(action: .loadDocumentData(boundingBoxConverter: documentController))
         }
 
-        if state.changes.contains(.annotations) {
+        if state.changes.contains(.annotations) && state.changes.contains(.selectionDeletion) {
```

Comment:
We can just leave `if state.changes.contains(.selectionDeletion) {` here.

Reply:
Check https://github.com/zotero/zotero-ios/pull/875/commits/e4f821f0e933122b71fd926b16252e8bf46f3f44